Library/Formula/poppler.rb

```diff
@@ -52,7 +52,9 @@ def install
       --enable-introspection=yes
     ]
 
-    if build.with? "qt"
+    if build.with?("qt") && build.with?("qt5")
+      odie "poppler: --with-qt and --with-qt5 cannot be used at the same time"
```

Comment:
I see. Would it be good to have an audit test that checks for uses of `odie` and suggests to use `raise` instead? Or would that be overkill? I picked `odie` because it was the first thing that came to my mind and I saw it used in multiple formulae for the exact same purpose (to fail on incompatible options) …